test/e2e_node/eviction_test.go

```diff
@@ -813,8 +813,9 @@ func runEvictionTest(f *framework.Framework, pressureTimeout time.Duration, expe
 					RestartPolicy: v1.RestartPolicyNever,
 					Containers: []v1.Container{
 						{
-							Image: imageutils.GetPauseImageName(),
-							Name:  podName,
+							Image:           imageutils.GetPauseImageName(),
+							Name:            podName,
+							ImagePullPolicy: "Always",
```

Comment:
Always seems a bit aggressive. I would expect `IfNotPresent` will work

Reply:
Looking at other images in eviction, they were using Always. If you want this, can we do this as one fix rather than one-offs?

Reply:
Here is the follow up. https://github.com/kubernetes/kubernetes/pull/133359